srcpkgs/m4ri/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'm4ri'
+pkgname=m4ri
+version=20200125
+revision=1
+build_style=gnu-configure
+configure_args="--enable-openmp"
+hostmakedepends=pkg-config
+makedepends=libpng-devel
+short_desc="Library for fast arithmetic with dense matrices over GF(2)"
+maintainer="Eloi Torrents <eloitorrentsj@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://bitbucket.org/malb/m4ri"
+distfiles="${homepage}/downloads/${pkgname}-${version}.tar.gz"
+checksum=0dfb34aed351882a0f2281535ea6f81c690a5efeb14edab131d9ba0dffe44863
+
+m4ri-devel_package() {
+    short_desc+=" - development files"
+    depends="${sourcepkg}>=${version}_${revision}"
+    pkg_install() {
+    	vmove usr/include
+	vmove "usr/lib/*.a"
+    	vmove "usr/lib/*.so"
+#    	vmove usr/lib/pkgconfig
```

Comment:
Uncomment this after you change the previous line so the `pkgconfig` goes into `m4ri-devel`.